airbyte-integrations/connectors/source-shopify-singer/README.md

```diff
@@ -0,0 +1,62 @@
+# Source Shopify Singer
+
+This is the repository for the Shopify source connector, based on a Singer tap.
+For information about how to use this connector within Airbyte, see [the User Documentation](https://docs.airbyte.io/integrations/sources/shopify).
+
+## Local development
+### Prerequisites
+**To iterate on this connector, make sure to complete this prerequisites section.**
+
+#### Build & Activate Virtual Environment
+First, build the module by running the following from the `airbyte` project root directory: 
+```
+./gradlew :airbyte-integrations:connectors:source-shopify-singer:build
+```
+
+This will generate a virtualenv for this module in `source-shopify-singer/.venv`. Make sure this venv is active in your
+development environment of choice. To activate the venv from the terminal, run:
+```
+cd airbyte-integrations/connectors/source-shopify # cd into the connector directory
+source .venv/bin/activate
+```
+If you are in an IDE, follow your IDE's instructions to activate the virtualenv.
+
+#### Create credentials
+**If you are a community contributor**, follow the instructions in the [documentation](https://docs.airbyte.io/integrations/sources/shopify)
+to generate the necessary credentials. Then create a file `secrets/config.json` conforming to the `source_shopify_singer/spec.json` file.
```

Comment:
is it relatively straight foward for a user to find all of these creds themselves or are there any docs to link to help them?

Reply:
This is non-trivial but described in `shopify.md`